src/client/client.c

```diff
@@ -549,6 +549,12 @@ int client_call_manager(Client *client) {
                 }
                 r = method_lifecycle_action_on(client, client->opargv[0], client->opargv[1], "ReloadUnit");
         } else if (streq(client->op, "monitor")) {
+                /* monitor node connection status changes */
+                if (client->opargc == 1 && streq(client->opargv[0], "node-connection")) {
```

Comment:
I am not super happy about the command `hirtectl monitor node-connection`, but since its continuously monitoring the node state the monitor cmd seems kind of like a good place. 
The `method_monitor_node_connection_state` code is encapsulated, so shuffling it around later isn't tedious.

Reply:
I agree, this seems like a good place for monitoring the nodes.